_includes/pair-programming.md

```diff
@@ -0,0 +1,60 @@
+# Pair programming
+
+## How to pair
+
+In pair programming, two programmers work together at one workstation.
+The **driver** types at the keyboard while the **navigator** directs
+the driver. Even though the driver is at the keys, their role is the
+more passive; the navigator directs the driver on what to type. The
+driver executes their commands.
+
+The foundation of pair programming is communication. Partners must
+always be communicating throughout. The navigator directs the driver,
+but the driver will often need clarification about what the navigator
+intends for them to do. When in disagreement, the pair needs to
+discuss the proper course of action.
+
+**ABS**: Always be switching!
+
+## Benefits of pair programming
+
+There are a number of reasons we pair:
+
+1. Pairs are a second pair of eyes. Since most novice mistakes are
+   trivial, your pair has a greater chance of immediately identifying
+   a problem that could have otherwise taken you a long time to
+   discover.
+2. Every student will learn different things outside of class; pairing
+   is a way to bring that knowledge back into the class and share it
+   with other students. This is one of the reasons we have new pairs
+  for every project; we want everyone to benefit from everyone else's
+  experience.
+3. It is important to maintain focus throughout the class day. Working
+   solo, it is easy to become distracted or zone out. Working in
+   pairs, students help keep each other honest.
+4. Working with other people is fun; we do a lot of coding, and it
+   ought to be enjoyable.
+
+## Common mistakes of pair programming
```

Comment:
Maybe instead of talking about mistakes, go more in-depth about what good pair programming looks like. Set expectations.